### PR TITLE
Create infrastructure with Terraform

### PR DESCRIPTION
**Pull request by dtrendiuk**

### CODE REVIEW

File: Terraform/user_data_phpmyadmin.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+cd /usr/share/nginx/html/
+sudo wget https://www.phpmyadmin.net/downloads/phpMyAdmin-latest-all-languages.tar.gz
+sudo mkdir phpmyadmin
+sudo tar -xvzf phpMyAdmin-latest-all-languages.tar.gz -C phpmyadmin --strip-components 1
+sudo systemctl restart mariadb
+
+###########################AWSLOGS#################################################################
+
+sudo yum install -y awslogs
+region=$(curl 169.254.169.254/latest/meta-data/placement/availability-zone | sed s'/.$//')
```
Comment (DmytroMorhulDevPro) [inline Terraform/user_data_phpmyadmin.sh]: You already took active region in terraform, why you asked it once again?
I guess you need to add one more variable to this script.

File: Terraform/user_data_webserver.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+sudo systemctl enable nginx
+sudo systemctl enable php-fpm.service
+sudo systemctl start nginx
+sudo systemctl start php-fpm.service
+
+###########################AWSLOGS#################################################################
+
+sudo yum install -y awslogs
+region=$(curl 169.254.169.254/latest/meta-data/placement/availability-zone | sed s'/.$//')
```
Comment (DmytroMorhulDevPro) [inline Terraform/user_data_webserver.sh]: the same here

**Review by DmytroMorhulDevPro — APPROVED** [review]

File: Terraform/user_data_phpmyadmin.sh (hunk 1 above)
Comment (dtrendiuk) [reply Terraform/user_data_phpmyadmin.sh]: Made changes in the [commit](https://github.com/dtrendiuk/learning/pull/1/commits/0d0f0cbcd91a96a1edf781282ab8334e4582384d)

File: Terraform/user_data_webserver.sh (hunk 2 above)
Comment (dtrendiuk) [reply Terraform/user_data_webserver.sh]: Made changes in the [commit](https://github.com/dtrendiuk/learning/pull/1/commits/0d0f0cbcd91a96a1edf781282ab8334e4582384d)

**Pull request merged**